uptrain/framework/base.py

```diff
@@ -21,6 +21,8 @@
 class Settings(BaseSettings):
     # uptrain stores logs in this folder
     logs_folder: str = "/tmp/uptrain_logs"
+    # uptrain stores static resources in this folder
+    resources_folder: str = "/tmp/uptrain_resources"
```

Comment:
@ananis25 any thoughts?

Reply:
this is not needed once I refactor GetSchemaDefinition

Reply:
yeah in general, we should create subdirectories inside the logs-folder. Keep configuration as small as possible.

Reply:
removed